src/components/settings/SettingsLMSTab/tests/BlackboardConfig.test.jsx

```diff
@@ -165,6 +168,11 @@ describe('<BlackboardConfig />', () => {
 
     expect(screen.getByText('Authorize')).not.toBeDisabled();
     userEvent.click(screen.getByText('Authorize'));
+
+    await act(async () => {
```

Comment:
We might be able to wrap `userEvent.click(screen.getByText('Authorize'));` within this act and it should do the thing you want?

Reply:
So apparently userEvents has built in act functionality and you're not supposed to wrap userEvents in act 😬 https://www.youtube.com/watch?v=MuRhQGl8lDU&t=1810s

Reply:
ty for sharing 🙇